<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To-Do List App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="additem_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>add list item here.</string>
    </property>
   </widget>
   <widget class="QListWidget" name="my_listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAllButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveItemButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Item</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddItemButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
